libnavigation-core/src/test/java/com/mapbox/navigation/core/accounts/BillingTest.kt

```diff
@@ -21,50 +20,57 @@ class BillingTest {
     private val applicationInfo: ApplicationInfo = mockk()
     private val metadata: Bundle = mockk()
 
-    @Before
-    fun setUp() {
-        every { appContext.packageManager } returns packageManager
-    }
-
     @After
     fun cleanUp() {
```

Comment:
Minor thing: same setup reasons from https://github.com/mapbox/mapbox-navigation-android/pull/2457#discussion_r377365230 apply to cleanup

Reply:
fixed!